components/Main/Hero.vue

```diff
@@ -1,17 +1,189 @@
 <script setup lang="ts">
+import { onMounted, onUnmounted } from 'vue';
```

Comment:
nuxt 3 uses a system names 'auto-imports' so for that is unnecesary import that type of things, i'll leave you the documentation anyway [nuxt](https://nuxt.com/docs/guide/concepts/auto-imports)

Reply:
hahahahahhahahahhahahhahahahhahahhahahahha ok I'll read it thanks